src/renderRepoCard.js

```diff
@@ -88,6 +97,7 @@ const renderRepoCard = (repo, options = {}) => {
       <style>
       .header { font: 600 18px 'Segoe UI', Ubuntu, Sans-Serif; fill: ${titleColor} }
       .description { font: 400 13px 'Segoe UI', Ubuntu, Sans-Serif; fill: ${textColor} }
+      .template { font: 400 13px 'Segoe UI', Ubuntu, Sans-Serif; fill: ${textColor}; font-weight: 500 }
       .gray { font: 400 12px 'Segoe UI', Ubuntu, Sans-Serif; fill: ${textColor} }
       .icon { fill: ${iconColor} }
       .archive-badge { font: 600 12px 'Segoe UI', Ubuntu, Sans-Serif; }
```

Comment:
Match the font style with the archive-badge please. 

rename that .archive-badge css rule to something generic like `badge` and use that for both template & archive

Reply:
okay. I will change

Reply:
done